django_cloud_deploy/integration_tests/cloudlib_test.py

```diff
@@ -1,88 +0,0 @@
-# Copyright 2018 Google LLC
```

Comment:
I added a new integration test for create_and_setup database workflow, which includes testing of cloud sql instance creation and database creation.